R/plot_with_settings.R

```diff
@@ -61,7 +61,7 @@ plot_with_settings_ui <- function(id) {
 #' Plot-with-settings module
 #'
```

Comment:
add `@name plot_with_settings`, the same way with `table_with_settings`?

Reply:
2d3c73b3db317fcf4886c21fac3c0e350b8c7aec